### DISCUSSION TITLE
Logistics Box not working

### DISCUSSION THREAD

**Discussion by SleekSycko** [Vdauphin/HeartsAndMinds#1148] [Question and answer] [answered]
When trying to acquire an item from the logistics box it give error: Line26 ((count _main_class) - 1) do { in core\fnc\log creat_load.sqf
is not a variable. How do I fix this? 

Sorry if easy answer. new to scripting.

**Comment by Vdauphin** [comment, marked as answer]
Hello,
You should try to the version from here https://github.com/Vdauphin/HeartsAndMinds/releases
Cheers

**Reply by SleekSycko** [reply]
Still getting the same error, boss.

**Reply by SleekSycko** [reply]
Am Suppose to define the main class? Just want to get this fixed so the supply mission is doable.

**Reply by Vdauphin** [reply]
Everything work out of the box, if you find a bug, create an issue

**Reply by SleekSycko** [reply]
Hey boss, fixed the issue. Not your fault at all. The server host I'm using did not give access to all the files within the PBO after reviewing the RTP. Thanks man awesome mission!.

**Reply by bluestorm383** [reply]
> Hey boss, fixed the issue. Not your fault at all. The server host I'm using did not give access to all the files within the PBO after reviewing the RTP. Thanks man awesome mission!.

Hey, Sleek! I'm having this same issue, how did you go about fixing it?
